testpar/t_shapesame.c

```diff
@@ -4319,8 +4282,6 @@ main(int argc, char **argv)
         return -1;
     }
 
-    mpi_rank_framework_g = mpi_rank;
```

Comment:
Instead of relying on parallel tests to set this global variable for the testframe.c framework somewhere in main, it gets passed to `TestInit()` now instead